task_processing/plugins/mesos/translator.py

```diff
@@ -2,17 +2,17 @@
 
 # https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto
 MESOS_TASK_STATUS_TO_EVENT = {
-    0: Event(platform_type='starting'),
-    1: Event(platform_type='running'),
-    2: Event(platform_type='finished'),
-    3: Event(platform_type='failed'),
+    0: Event(platform_type='starting', terminal=False),
```

Comment:
| should None mean anything different from False then?

You shouldn't be able to do ``None``? maybe I can provide a predicate like: ```lambda x: x in ['True', 'False']``` if the typing isn't strong enough.

|  if it doesn't - then let's at least make the field mandatory?
agreed